### PR TITLE
Added support of clear button on media field in shortcode generator.

### PR DESCRIPTION
**Pull request by dhaval-parekh**
Reference : Issue 1343

### CODE REVIEW

File: laterpay/asset_sources/js/laterpay-backend-shortcode-generator.js [hunk 1]
```diff
@@ -113,24 +113,92 @@
...
+             * To generate media preview mark for media field in shortcode generator.
+             *
+             * @param {string} target_field ID of target field.
+             *
+             * @returns {string} HTML markup for media preview.
+             */
+            get_media_markup: function ( target_field ) {
+
+                // @codingStandardsIgnoreStart
+                return '<div id="preview_' + target_field + '" class="lp-media-preview" data-target_field="' + target_field + '">' +
```
Comment (thrijith) [inline laterpay/asset_sources/js/laterpay-backend-shortcode-generator.js]: Does the attribute accept html string only or is jQuery object valid?
If jQuery object is valid maybe create the elements and send it.

File: laterpay/asset_sources/js/laterpay-backend-shortcode-generator.js [hunk 2]
```diff
@@ -113,24 +113,92 @@
...
+                       '<img class="preview-image" src="' + laterpay_shortcode_generator_labels.preview_images.gallery + '"/>' +
```
Comment (thrijith) [inline laterpay/asset_sources/js/laterpay-backend-shortcode-generator.js]: Add a `// jshint ignore:line` for longer lines. Try running `gulp js-build` to see the warnings.

File: laterpay/asset_sources/js/laterpay-backend-shortcode-generator.js (hunk 1 above)
Comment (dhaval-parekh) [reply laterpay/asset_sources/js/laterpay-backend-shortcode-generator.js]: We have to  return HTML string. I tried create element with JS but that didn't work.

File: laterpay/asset_sources/js/laterpay-backend-shortcode-generator.js (hunk 2 above)
Comment (dhaval-parekh) [reply laterpay/asset_sources/js/laterpay-backend-shortcode-generator.js]: Done.

**Pull request closed without merging**